src/main/scala-2.11/markets/orders/orderings/PriceOrdering.scala

```diff
@@ -18,18 +18,20 @@ package markets.orders.orderings
 import markets.orders.Order
 
 
+/** Trait defining price ordering over various subclasses of [[markets.orders.Order `Order`]]. */
 trait PriceOrdering[T <: Order] extends Ordering[T] {
 
-  override def compare(order1: T, order2: T): Int = {
+  def compare(order1: T, order2: T): Int = {
     if (hasPricePriority(order1, order2)) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using case matching instead of else if blocks](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1508311816/issues/source?bid=2921361&fileBranchId=3005746#l25)